annotation/load_generations.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import json
+import os
+import re
+import django
+import requests
+import click
+# kirubarajan: django model imports at bottom since you have to configure the environment first
+
+MIN_LENGTH = 10
+
+
+# helper function to fix malformatted JSON
+def _clean_json(string):
+    string = re.sub(",[ \t\r\n]+}", "}", string)
+    string = re.sub(",[ \t\r\n]+\]", "]", string)
+    return string
+
+
+@click.command()
+@click.option('--playlist_name', help='Name of new playlist to create.')
+@click.option('--generations', help='JSON file containing generations.')
+@click.option('--prompts', help='JSON file containing prompts.')
+def populate_db(playlist_name, generations, prompts):
+    # open saved generations and parse JSON
+    click.echo("Loading generations...")
+
+    with open(generations) as file:
+        generation_fixture = json.loads(_clean_json(file.read()))
+    with open(prompts) as file:
+        prompt_fixture = json.loads(_clean_json(file.read()))
+    
+    # creating system
+    model_name = generation_fixture['generation-model']
+    system = System.objects.create(
+        name=model_name   
+    )
+
+    # creating dataset
+    dataset_name = generation_fixture['dataset']
+    dataset_split = generation_fixture['split']
+    dataset = Dataset.objects.create(
+        name=dataset_name,
+        split=dataset_split
+    )
+
+    # creating playlist
+    playlist = Playlist.objects.create(
```

Comment:
Same question applies here. 

I think the behaviour we'd like is for this code to reuse the existing Dataset/Playlist/etc. if they already exist.

Reply:
Agreed